dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -55,6 +55,12 @@
               </div>
 
               <div v-if="!pageScrolled" id="createButtonAndPageTitle" class="flex align-content-end align-items-center">
+                <PrimeButton
```

Comment:
As this button is used here and in FrameworkDataSearchResult, can it be extracted as a component, which than can be imported wherever needed. This would lower code duplication and make it more maintainable.

Reply:
Also the symbol in the button should be a "plus" instead of a ?.
![image](https://github.com/d-fine/Dataland/assets/104447806/cc74bf22-9f41-45b9-8286-b37f3c0581e9)

Reply:
I also saw in the Figma that the request button is designed with a plus icon. However, it seems like the "New Dataset" button currently occupies this icon. Moreover, the Figma indicates that the "New Dataset" button should only be displayed on the "MY DATASETS" page whereas we decided to still provide the possibility to create a new dataset on the "AVAILABLE DATASETS" page as it was before this story. Hence, I needed to search for a new symbol that can be used as an icon and thought the question mark would maybe be the most suitable one as it indicates asking somebody for something, in our case a dataset.

Reply:
The first comment is implemented, RequestDataButton is now a component on its own